.teamcity/src/main/kotlin/common/extensions.kt

```diff
@@ -160,7 +160,7 @@ fun BuildType.paramsForBuildToolBuild(buildJvm: Jvm = BuildToolBuildJvm, os: Os,
         param("env.GRADLE_CACHE_REMOTE_USERNAME", "%gradle.cache.remote.username%")
 
         param("env.JAVA_HOME", javaHome(buildJvm, os, arch))
-        param("env.GRADLE_OPTS", "-Xmx1536m -XX:MaxPermSize=384m")
```

Comment:
Should this use MaxMetaspace instead?

Reply:
`MaxPermSize` hasn't been doing anything since 8, and therefore for however long Gradle has been building on either 8 or 11, so I don't think we actually need this parameter.